platforms/flink/Dockerfile

```diff
@@ -0,0 +1,55 @@
+ARG FLINK_VERSION=1.20
+
+FROM flink:${FLINK_VERSION} AS build_base
+
+ARG FLINK_VERSION=1.20
+ARG KAFKA_CONNECTOR_VERSION=3.4.0
+ARG PYTHON_VERSION=3.11
+
+# The upstream flink image does not contain the JDK but the JRE only.
+# In order to run pyflink we need the JDK ... so we get a JDK.
```

Comment:
This is needed because we need a jdk to install pyflink and the standard image does not have it installed. We need to install pyflink here so that we can deploy the flink python application from inside the container.